test/integration/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "testDependencies": {
+        "selenium-webdriver": "2.44.0"
```

Comment:
Now that this is separate from `package.json`, maybe we should also include `chromedriver`, so that can be tracked (until we go with another way of doing it).